H/array_ops.h

```diff
@@ -34,7 +34,7 @@ typedef struct {
 typedef struct {
   OPDS    h;
   ARRAYDAT* arrayDat;
-  MYFLT   *isizes[VARGMAX];
+  MYFLT   *iargs[VARGMAX];
```

Comment:
Curious, why the change from isizes to iargs?

Reply:
It's a mistake, it's reverted back to how it was.